mayastor-test/test_csi.js

```diff
@@ -111,7 +112,7 @@ function getFsType(mp) {
 }
 
 describe('csi', function() {
-  this.timeout(10000); // for network tests we need long timeouts
+  this.timeout(60000); // for network tests we need long timeouts
```

Comment:
60s is a bit too much. Why did you change this?

Reply:
I am happy to try smaller values, but 10sec on my system fails the test with timeout.  Given that this is basically to guard against tests that hang, what is the harm in using 1min?  If you envisage a test case that has to take less than a particular amount of time to be considered a pass, then that can surely be asserted explicitly by that test.  This is better that having an entire test suite that relies on a performant system.

Reply:
I tried reverting this on my new setup (running on the host, no VM) and the tests passed!
(I should have tried doing this in the first place, when I got rid of the VM - it was obvious that everything was running faster)  so I am reverting this.